snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/QueryRefSetUpdateRequestBuilder.java

```diff
@@ -40,7 +41,7 @@ public QueryRefSetUpdateRequestBuilder setReferenceSetId(String refSetId) {
 	}
 	
 	public QueryRefSetUpdateRequestBuilder setSource(Map<String, Object> source) {
-		return setModuleId((String) source.get("moduleId")).setReferenceSetId((String) source.get("referenceSetId"));
+		return setModuleId((String) source.get(SnomedRf2Headers.FIELD_REFSET_ID)).setReferenceSetId((String) source.get(SnomedRf2Headers.FIELD_REFSET_ID));
```

Comment:
`setModuleId` should use `FIELD_MODULE_ID` from `source`, not `FIELD_REFSET_ID`.